react-native-libraries.json

```
@@ -9545,5 +9545,17 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/dozsolti/react-native-use-list",
+    "npmPkg": "react-native-use-list",
+    "nameOverride": "React Native useList",
+    "examples": [
+      "https://github.com/dozsolti/react-native-use-list/blob/main/example/examples/FlatList/Pagination.tsx"
```

```suggestion
      "https://github.com/dozsolti/react-native-use-list/tree/main/example"
```

Usually, we link users to the example app root.